src/assets/scripts/client/math/core.js

```diff
@@ -236,4 +236,18 @@ export const generateRandomOctalWithLength = (length = 1) => {
     }
 
     return leftPad(value.join(''), length);
+};
+
+/**
+ * Generates a random number, on an (approximate) bell curve.
+ * AKA - Normal Distribution, Gaussian Distribution
+ * 
+ * @function getNormalDistributedNumber
+ * @param average
+ * @return {number}
+ */
+export const getNormalDistributedNumber = (average) => {
```

Comment:
try to avoid the word `get` here.  again, like I mentioned above, the word `calculate` would be more appropriate.

this should also be a normal function instead of an arrow function: 
```javascript
export function getNormalDistributedNumber(average) {
 // ...
}
```